docs/install-alternative.md

```diff
@@ -17,19 +17,81 @@ Follow the instructions below, or [watch a video by Drupalize.me](https://youtu.
         # Install master branch as a git clone. Great for contributing back to Drush project.
         composer global require drush/drush:dev-master --prefer-source
 
-* Alternate way to install for all users via Composer:
-
-        COMPOSER_HOME=/opt/drush COMPOSER_BIN_DIR=/usr/local/bin COMPOSER_VENDOR_DIR=/opt/drush/7 composer require drush/drush:7
-
 * [Documentation for composer's require command.](http://getcomposer.org/doc/03-cli.md#require)
 * Uninstall with : `composer global remove drush/drush`
 
+Install Drush for all users via Composer
+------------
+If you need a common Drush install available for all users on a system, follow the composer install steps below.
+
+1. [Install Composer globally][composer install global].
+1. Create and/or navigate to a directory path for the single composer drush install. In this example we are going to install drush version 7.x
+
+        sudo mkdir --parents /opt/drush/7.x
+        cd /opt/drush/7.x
+
+1. Initialise a new composer project that requires drush. Here you may also specify the drush [package version][composer package version] you wish composer to install. In this example we specify drush version 7.x with the [package version][composer package version] string `7.*`
+
+        sudo composer init --require=drush/drush:7.* -n
+
+1. Configure the path composer should use for the drush package's [vendor binaries][composer vendor binaries] or command-line scripts. Choose a directory path that is used in the `$PATH` configuration for all users, for example `/usr/local/bin`:
+
+        sudo composer config bin-dir /usr/local/bin
+
+1. Now run the `composer install` command. A `composer.json` file containing each of the configurations we just did has been created.
+
+        sudo composer install
+
+    The `composer.json` file tells composer to install (or update) drush at the [package version][composer package version] we specified and to put the [vendor binaries][composer vendor binaries] where all users can access them.
+
+1. Lastly, if you are using bash with command completion you can enable drush completion for all users by simply symlinking the `drush.complete.sh` shell script into the correct location.
+
+        sudo ln -s /usr/local/bin/drush.complete.sh \
+          /etc/bash_completion.d/drush
+
+**Important Tip:** When installing *drush 6.x* there are *dependencies not managed by composer* that will require download on first run. You
```

Comment:
Drush 6 is no longer supported. Should we still document the installation procedure for it?